components/StyledInputAmount.js

```diff
@@ -187,7 +227,7 @@ StyledInputAmount.propTypes = {
   /** To enable the currency picker */
   hasCurrencyPicker: PropTypes.bool,
   /** A list of currencies presented in the currency picker */
-  availableCurrencies: PropTypes.arrayOf(PropTypes.string),
+  availableCurrencies: PropTypes.object,
```

Comment:
In terms of developer experience, this makes it more complex to use the component and we're duplicating code. Can we keep accepting a list of currency and let `StyledInputAmount` be in charge of the internationalization?